src/style.css

```diff
@@ -2,0 +1,44 @@
+* {
+  margin: 0;
+}
+
+body {
+  width: 96px;
+  height: 116px;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.stars-block {
+  display: flex;
+  flex-direction: row-reverse;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  display: block;
+  background-image: url(images/star.svg);
+  background-size: cover;
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(n + 5) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(n + 4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-child(n + 3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(n + 2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-child(n) {
+  background-image: url(images/star-active.svg);
```

Comment:
The CSS for the `stars--N` modifiers is incorrect. The current implementation activates the wrong stars. Ensure that the block with `stars--N` modifier activates exactly the first `N` stars. For example, `.stars--1` should activate the first star, `.stars--2` should activate the first two stars, and so on.